code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -698,8 +698,7 @@
 
 	var/mob/living/victim = M
 	do_attack_animation(M)
-	var/armor_block = victim.get_soft_armor("bio", BODY_ZONE_CHEST)
-	victim.apply_damage(100, STAMINA, BODY_ZONE_CHEST, armor_block) //This should prevent sprinting
+	victim.apply_damage(victim.modify_by_armor(100, BIO, def_zone = BODY_ZONE_HEAD), STAMINA, BODY_ZONE_HEAD) //This should prevent sprinting
```

Comment:
Why modify by armor explicitly instead of leaving it to the apply damage call?

Reply:
whoops, probably did it after doing the nasty limb damage ones above.